crime-application-tracking/src/test/java/uk/gov/justice/laa/crime/application/tracking/integration/ApplicationTrackingServiceTest.java

```diff
@@ -0,0 +1,260 @@
+package uk.gov.justice.laa.crime.application.tracking.integration;
+
+import okhttp3.mockwebserver.MockWebServer;
+import org.junit.jupiter.api.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+import uk.gov.justice.laa.crime.application.tracking.CrimeApplicationTrackingApplication;
+import uk.gov.justice.laa.crime.application.tracking.model.ApplicationTrackingOutputResult;
+import uk.gov.justice.laa.crime.application.tracking.model.ApplicationTrackingOutputResult.RequestSource;
+import uk.gov.justice.laa.crime.application.tracking.testutils.FileUtils;
+import uk.gov.justice.laa.crime.application.tracking.testutils.JsonUtils;
+
+import java.io.IOException;
+
+import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.DEFINED_PORT;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest(classes = CrimeApplicationTrackingApplication.class, webEnvironment = DEFINED_PORT)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class ApplicationTrackingServiceTest {
+    private MockMvc mvc;
+
+    private static MockWebServer mockWebServer;
+
+    private ApplicationTrackingOutputResult applicationTrackingOutputResultJson;
+
+    @Autowired private WebApplicationContext webApplicationContext;
+
+    @BeforeAll
+    public void initialiseMockWebServer() throws IOException {
+        mockWebServer = new MockWebServer();
+        mockWebServer.setDispatcher(MockWebServerStubs.forDownstreamApiCalls());
+        mockWebServer.start(9999);
+    }
+
+    @AfterAll
+    protected void shutdownMockWebServer() throws IOException {
+        mockWebServer.shutdown();
+    }
+
+    @BeforeEach
+    public void setup() {
+        String content = FileUtils.readFileToString("testdata/atsrequest.json");
+        applicationTrackingOutputResultJson = JsonUtils.jsonToObject(content, ApplicationTrackingOutputResult.class);
+
+        this.mvc = MockMvcBuilders.webAppContextSetup(this.webApplicationContext).build();
+    }
+
+    @Test
+    void givenCreateApplicationRequest_shouldCreateAudit_andDoRequiredUpdates() throws Exception {
+        String string = createApplicationTrackingOutputResult(RequestSource.CREATE_APPLICATION);
```

Comment:
Can you update the names of all the instances of
`String string`
to something like
`String content `
like above